### ISSUE TITLE
container_16 and container_24

### ISSUE THREAD

**Issue by Ybbet** [nathansmith/adapt#20]
Is it possible to consider the variation of Adapt.js for container_16 and container_24?

**Comment by nathansmith** [comment]
I hadn't made container_16 or container_24 versions, because (aside from the width of 960 pixels), those don't divide evenly along the same proportions as container_12.

However, you can generate your own via this tool…

http://grids.heroku.com

---

Related: I'm currently working on a fluid grid system that (I hope) will obviate the need for 960 altogether…

http://host.sonspring.com/fluid-grid/

As an "in the wild" example, my sister's "save the date" engagement site…

http://tedbethany.com

Basically, the code for that looks like…

```
<div class="width-50 mobile-width-33">…</div>
<div class="width-25 mobile-width-33">…</div>
<div class="width-25 mobile-width-33">…</div>
```
